source/manual/business-readiness-publish-changes.html.md

```diff
@@ -11,18 +11,13 @@ review_in: 3 months
 
 *If you are trying to update the content returned by the business readiness finder, see: [Update content for the business readiness finder](/manual/business-readiness-update-content.html). For more information about the finder itself, see: [EU exit business readiness finder](/manual/business-readiness-finder.html)*
 
-The full journey for the  [business readiness finder][business-readiness-finder] is made up of the following content items:
+The content for the start page ([`/business-uk-leaving-eu`][business-readiness-finder]) can be updated in Mainstream Publisher as you would any other start page.
```

Comment:
The link for the start page should be to: https://www.gov.uk/business-uk-leaving-eu